sensor_msgs/Image image
---
hsr_navigation/Labels labels
hsr_navigation/Points2D points